src/propcache/_helpers_c.pyx

```diff
@@ -1,5 +1,13 @@
 # cython: language_level=3
-from types import GenericAlias
+import sys
+import types
+
+if sys.version_info >= (3, 9):
+    GenericAlias = types.GenericAlias
+else:
+
+    def GenericAlias(cls):
+        return cls
```

Comment:
Wow, that's a sneaky one! Any ideas of how to produce coverage here?

Reply:
I tested it manually, I figure we are going to drop py3.8 in a few days/hours so there is no risk of it breaking before we remove it